partcad/src/partcad/project_factory.py

```
@@ -26,12 +26,16 @@
                 raise ValueError("Import configuration type is not set")
         self.import_config_type = config_obj.get("type")
         self.import_config_is_root = config_obj.get("isRoot", False)
+        self.import_config_url = config_obj.get("url")
         self.include_paths = self.config_obj.get("includePaths", [])
+        self.inherited_config = config_obj.get("inheritedConfig", {})
+        if parent and not self.inherited_config:
+            self.inherited_config = {"manufacturable": parent.is_manufacturable}
 
 
 class ProjectFactory(ImportConfiguration):
     def __init__(self, ctx, parent, import_config_obj):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)